webpack/assets/javascripts/react_app/redux/reducers/hosts/storage/vmware.js

```diff
@@ -123,7 +123,7 @@ export default (state = initialState, { type, payload }) => {
       });
     case STORAGE_VMWARE_DATASTORES_SUCCESS:
       return state
-        .set('datastores', payload.results)
+        .set('datastores', response.results)
         .set('datastoresLoading', false);
     case STORAGE_VMWARE_DATASTORES_FAILURE:
       return state.set('datastoresError', payload.error.message);
```

Comment:
should this be ?
```suggestion
       return state.set('datastoresError', reponse.error.message);
```
I think there are some other points here where you need to switch the two